app/dashboard/embed.py

```diff
@@ -218,9 +219,10 @@ def embed(request):
             i += 1
             value_eth = str(round(bounty.value_in_eth/10**18, 2)) + "ETH" if bounty.value_in_eth else ""
             value_in_usdt = str(round(bounty.value_in_usdt, 2)) + "USD" if bounty.value_in_usdt else ""
+            eth_value_in_usdt = str(round(get_eth_to_usdt(), 2)) + "/ETH"
             value_native = "{} {}".format(round(bounty.value_true, 2), bounty.token_name)
 
-            value = "{}, {}".format(value_eth, value_in_usdt)
+            value = "{}, (${} @ {})".format(value_eth, value_in_usdt, eth_value_in_usdt)
```

Comment:
> (${} @ {})

there are no labels here, right?  i think `(${} @ {} ETH/USD)` is better UX

Reply:
Should everything look like:  <i>(201.0 USD @ 700 USD/ETH)</i>? I indended to have them all in the format suggested in the issue as follows: <i>(201.0 USD @ $700/ETH)</i>.

Reply:
Nope, I don't think it's verbose. Just wanted to clarify is we should use $ verse USD. Currently, the USD value is displayed with the $ and just wanted to make sure I keep everything consistent. So I actually misrepresented the above and it should look as follows:

'''
($201.0 @ 700 USD/ETH)
'''

I will implement as shown above.